packages/lesspipe/build.sh

```diff
@@ -21,4 +22,5 @@ termux_step_post_make_install() {
 	mkdir -p "$TERMUX_PREFIX"/etc/profile.d
 	echo "export LESSOPEN='|$TERMUX_PREFIX/bin/lesspipe.sh %s'" \
 		> "$TERMUX_PREFIX"/etc/profile.d/lesspipe.sh
+	ln -sf $PREFIX/bin/lesspipe.sh $PREFIX/bin/lesspipe
```

Comment:
It is better to use `TERMUX_PREFIX` rather than `PREFIX`.

Reply:
In the build scripts definitely.

Reply:
Done